Sources/HaCWebsiteLib/Controllers/EventApiController.swift

```diff
@@ -0,0 +1,103 @@
+import Kitura
+import KituraNet
+import SwiftyJSON
+import Foundation
+import DotEnv
+
+struct EventApiController {
```

Comment:
I like that we're coming up with this whole API thing to start off with. It's going to make website maintenance much easier and less error prone.